rqt_joint_trajectory_controller/src/rqt_joint_trajectory_controller/joint_limits_urdf.py

```diff
@@ -33,12 +33,15 @@ def get_joint_limits(key='robot_description', use_smallest_joint_limits=True):
             if jtype == 'fixed':
                 continue
             name = child.getAttribute('name')
+            try:
+                limit = child.getElementsByTagName('limit')[0]
+            except:
+                continue
```

Comment:
I see two options:
1. just do not issue warnings for revolute and prismatic joints, might spam though
2. defer the skipping to https://github.com/ros-controls/ros_controllers/blob/rqt-jtc-improvements/rqt_joint_trajectory_controller/src/rqt_joint_trajectory_controller/joint_trajectory_controller.py#L246

Maybe it is not worth the effort.